ExtraDry/ExtraDry.Blazor/Components/Dry/DryStaticFilter.razor

```diff
@@ -0,0 +1,3 @@
+﻿@namespace ExtraDry.Blazor
+
+<div style="display: none" data-initial-level="2" />
```

Comment:
If the user overrides this value, it should probably be what the user has input, e.g. @@InitialHierarchyLevel.

Reply:
Fixed